src/main/java/org/folio/config/ApplicationConfig.java

```diff
@@ -105,6 +109,36 @@
 @ComponentScan({ "org.folio" })
 public class ApplicationConfig {
 
+  private static final Logger LOGGER = LogManager.getLogger();
+
+  @Value("${KAFKA_HOST:kafka}")
+  private String kafkaHost;
+  @Value("${KAFKA_PORT:9092}")
+  private String kafkaPort;
+  @Value("${OKAPI_URL:http://okapi:9130}")
+  private String okapiUrl;
+  @Value("${REPLICATION_FACTOR:1}")
```

Comment:
is `REPLICATION_FACTOR` exposed in any other folio module? is it related to kafka?

Reply:
yes. we used this parameter in all our modules related to Kafka

Reply:
I think it would be good if it is called `KAFKA_REPLICATION_FACTOR`.

Reply:
For this, we should rename it on our other modules.